docs/programming-languages.md

```diff
@@ -50,11 +50,24 @@ This brings some advantages, like programs starting significantly faster than
 regular Python, but comes at the cost at not being fully compatible with code
 written for regular Python.
 
+MicroPython works as a different Python interpreter, rather than as a library.
+It can be invoked by running it from the command line, for example:
+`micropython -c "import sys; print(sys.version)"` or
+`micropython -c "from ev3dev2.sound import Sound; Sound().speak('hello world')"`.
+You may add this as the first line of your Python script to run it with MicroPython:
+`#!/usr/bin/env micropython`.
+
 [Pybricks] is a specialized version of MicroPython for LEGO MINDSTORMS and LEGO
 Powered Up smart hubs. Pybricks v2.0 is used by LEGO Education as their [Python
 for EV3](https://education.lego.com/en-us/product-resources/mindstorms-ev3/teacher-resources/python-for-ev3)
 solution and is the same version that is available in ev3dev-stretch.
 
+You may run Pybricks from command line like this:
+`brickrun -r -- pybricks-micropython -c "from pybricks.hubs import EV3Brick; EV3Brick().speaker.beep()"`.
+You may add this as the first line of your Python script,
+`#!/usr/bin/env pybricks-micropython`,
+and then run it from the brick's file browser to run it with Pybricks.
+
```

Comment:
Maybe worth mentioning `pybricks-repl` here too?

Reply:
I tried it just now. It seems the `pybricks-repl` is the `pybricks-micropython` in interactive mode, isn't it?

I have been a developer using standard Python for years now, and I hardly find [standard Python's interactive mode](https://docs.python.org/3/tutorial/interpreter.html#interactive-mode) useful. Those `>>>` usually make copy-and-paste miserable, and it is not good for demonstrating a longer code snippet. For one-liners, I would rather use `python -c "print('hello world')"`.

That being said, I can send one more commit mentioning `pybricks-repl`. Stay tuned.